server/lib/configuration.js

```diff
@@ -30,6 +30,14 @@ const conf = module.exports = convict({
     env: 'ALLOWED_PARENT_ORIGINS',
     format: Array
   },
+  amplitude: {
+    disabled: {
+      default: false,
+      doc: 'Disable amplitude events',
+      env: 'DISABLE_AMPLITUDE',
```

Comment:
Could you rename the environment variable to `AMPLITUDE_DISABLED` for consistency with other environment variable names?

Reply:
@shane-tomlinson sure.